js/app.js

```diff
@@ -52,6 +52,93 @@ var Textile = {
     },
     parseZip: function(zip) {
         console.log("parsing: zip data");
+        if (zip.file("html/photos.htm")) {
+            Textile.parseHtml(zip);
+        }
+        else {
+            Textile.parseJson(zip);
+        }
+    },
+    parseJsonYourPosts: function(mapper) {
+        var status_updates = mapper["status_updates"];
+        status_updates.forEach(function (status_update) {
+            var milliUnix = 1000 * status_update["timestamp"];
+            var attachments = status_update["attachments"];
+            var isoDate = new Date(milliUnix).toISOString();
+            if (attachments) {
+                var statusFileCount = attachments.length;
+                var msg = " files in a status on ";
+                console.log(statusFileCount + msg + isoDate);
+            }
+            return;
+        });
+        return true
+    },
```

Comment:
⌛️For my purposes, add Textile.parseJsonYourPosts
⏳For this PR, remove Textile.parseJsonYourPosts

Reply:
This is now irrelevant for this PR.